lti_consumer/lti.py

```diff
@@ -83,14 +85,14 @@ def parse_result_json(json_str):
                 log.error("[LTI] %s", msg)
                 raise LtiError(msg)
         except (TypeError, ValueError) as err:
-            msg = "Could not convert resultScore to float: {}".format(err.message)
+            msg = "Could not convert resultScore to float: {}".format(str(err))
             log.error("[LTI] %s", msg)
             raise LtiError(msg)
 
     return score, json_obj.get('comment', "")
 
 
-class LtiConsumer(object):
+class LtiConsumer:  # pylint: disable=bad-option-value, old-style-class
```

Comment:
You shouldn't remove this just yet because the code will need to run in both python2 and python3 for a little while so it should stay compatible with both.